stages/apps/sharedsvc/sonarqube.tf

```diff
@@ -0,0 +1,22 @@
+data "vault_generic_secret" "sonarqube" {
+  path = "lead/aws/265560927720/sonarqube"
```

Comment:
we should derive the account number from the [`caller_identity` data source](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity)

Reply:
The hardcoding was intentional; as we didn't see a reason to make it dynamic, but I'm fine making it dynamic.